assets/xml/objects/object_zod.xml

```diff
@@ -1,50 +1,72 @@
-        <Limb Name="object_zod_Standardlimb_00D604" Type="Standard" EnumName="OBJECT_ZOD_LIMB_06" Offset="0xD604" />
-        <Limb Name="object_zod_Standardlimb_00D610" Type="Standard" EnumName="OBJECT_ZOD_LIMB_07" Offset="0xD610" />
-        <Limb Name="object_zod_Standardlimb_00D61C" Type="Standard" EnumName="OBJECT_ZOD_LIMB_08" Offset="0xD61C" />
-        <Limb Name="object_zod_Standardlimb_00D628" Type="Standard" EnumName="OBJECT_ZOD_LIMB_09" Offset="0xD628" />
-        <Skeleton Name="object_zod_Skel_00D658" Type="Flex" LimbType="Standard" LimbNone="OBJECT_ZOD_LIMB_NONE" LimbMax="OBJECT_ZOD_LIMB_MAX" EnumName="ObjectZodLimb" Offset="0xD658" />
-        <Animation Name="object_zod_Anim_00D9B0" Offset="0xD9B0" />
+        <!-- Tijo Animations -->
+        <Animation Name="gTijoPlayingVivaceAnim" Offset="0x2E8" />
+        <Animation Name="gTijoReadyToPlayAnim" Offset="0x894" />
+        <Animation Name="gTijoArmsFoldedAnim" Offset="0xA9C" />
+        <Animation Name="gTijoPlayingLentoAnim" Offset="0xD94" />
+
+        <!-- Tijo DisplayLists -->
+        <DList Name="gTijoLowerBodyChairDL" Offset="0x39A0" />
+        <DList Name="gTijoUpperBodyDL" Offset="0x3DA0" />
+        <DList Name="gTijoUpperRightArmDL" Offset="0x4018" />
+        <DList Name="gTijoLowerRightArmDL" Offset="0x4130" />
+        <DList Name="gTijoRightHandDL" Offset="0x4258" />
+        <DList Name="gTijoUpperLeftArmDL" Offset="0x43E8" />
+        <DList Name="gTijoLowerLeftArmDL" Offset="0x4500" />
+        <DList Name="gTijoLeftHandDL" Offset="0x4628" />
+        <DList Name="gTijoHeadDL" Offset="0x47B8" />
+
+        <!-- Tijo Textures /> -->
+        <Texture Name="gTijoSkinTex" OutName="tijo_skin" Format="rgba16" Width="8" Height="8" Offset="0x4B50" />
+        <Texture Name="gTijoChairTex" OutName="tijo_chair" Format="rgba16" Width="32" Height="32" Offset="0x4BD0" />
+        <Texture Name="gTijoHandTex" OutName="tijo_hand" Format="rgba16" Width="16" Height="16" Offset="0x53D0" />
+        <Texture Name="gTijoNoseTex" OutName="tijo_nose" Format="rgba16" Width="8" Height="8" Offset="0x55D0" />
+        <Texture Name="gTijoSkinSpotsTex" OutName="tijo_skin_spots" Format="rgba16" Width="32" Height="32" Offset="0x5650" />
+        <Texture Name="gTijoEyesOpen" OutName="tijo_eyes_open" Format="rgba16" Width="32" Height="32" Offset="0x5E50" />
+        <Texture Name="gTijoEyesHalfOpen" OutName="tijo_eyes_half_open" Format="rgba16" Width="32" Height="32" Offset="0x6650" />
+        <Texture Name="gTijoEyesClosed" OutName="tijo_eyes_closed" Format="rgba16" Width="32" Height="32" Offset="0x6E50" />
+        <Texture Name="gTijoMouthClosedTex" OutName="tijo_mouth_closed" Format="rgba16" Width="32" Height="32" Offset="0x7650" />
+        <Texture Name="gTijoMouthOpenTex" OutName="tijo_mouth_open" Format="rgba16" Width="32" Height="32" Offset="0x7E50" />
+
+        <!-- <Blob Name="object_zod_Blob_008650" Size="0x2800" Offset="0x8650" /> -->
```

Comment:
The size is wrong now, but actually this looks very like an unused set of vtx to me.